app/views/schools/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1 class="govuk-heading-xl">New school</h1>
+
+<%= form_with model: @school ||= School.new, url: schools_path, local: true do |form| %>
+  <% if @school&.errors&.any? %>
+    <%= form.govuk_error_summary %>
+  <% end %>
+  <%= form.govuk_text_field :name %>
+
+  <%= form.govuk_date_field :opened,
+                            legend: { text: "Enter the date the school was opened" },
+                            hint: { text: "For example, 31 3 1980" }
+  %>
+
+  <%= form.govuk_collection_radio_buttons :school_type,
+                                          School::SCHOOL_TYPES,
+                                          ->(option) { option },
+                                          ->(option) { option },
+                                          legend: { text: "What type of school is it?" }
```

Comment:
I imagine the two functions are one for value and one for text that option shows?

Reply:
Also, the massive white space, uuh, can't say I'm a fan :P But I appreciate it might be rubocop making you do it.

Reply:
> I imagine the two functions are one for value and one for text that option shows?

Yeah, that's right. For something more complex, we might want something more like the example in [the docs ](https://govuk-form-builder.netlify.app/form-elements/radios/):
```
departments = [
  OpenStruct.new(id: 1, name: 'Sales'),
  OpenStruct.new(id: 2, name: 'Marketing'),
  OpenStruct.new(id: 3, name: 'Finance')
]
```
and
```
<%= f.govuk_collection_radio_buttons :new_department_id, departments, :id, :name, legend: { text: 'Which department do you work for?' }, hint: { text: 'There should be a sign near your desk' } %>
```

Reply:
As for the whitespace, we could have something like
```
  <%= form.govuk_collection_radio_buttons(
        :school_type,
        School::SCHOOL_TYPES,
        ->(option) { option },
        ->(option) { option },
        legend: { text: "What type of school is it?" }
      )
  %>
```